docs/gathered-data.md

```diff
@@ -979,6 +964,23 @@ gathers the alerts that have been silenced.
   - 4.10+
 
 
+## StorageCluster
+
+collects `storageclusters.ocs.openshift.io` resources
+from Openshift Data Foundation Stack.
+
+API Reference:
+
+	https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+
+* Location in archive:
```

Comment:
@tremes @jholecek-rh I'm trying to find a nice way to document the path changes. Suggestions?

Reply:
The suggestion below looks good to me. Question is how it's rendered in the markdown.

Reply:
Looks good assuming we will not backport the change?

Reply:
@jholecek-rh Yes. If we backport it, we are going just to update the documentation the path to the backported version.